src/vm.nit

```diff
@@ -377,6 +390,37 @@ class VirtualMachine super NaiveInterpreter
 
 		return i != initialization_value
 	end
+
+	# Redef to add the numbering of variables and arguments
+	redef fun initialize_frame(node, mpropdef, args)
+	do
+		var f = new VmFrame(node, mpropdef, args)
+
+		# If this Frame is for a method then number variables into the body of the method
+		if node isa AMethPropdef then
```

Comment:
This is strange but it works perfectly.

Maybe the reason is around accessor simulation ?